plugins/module_utils/args_common.py

```diff
@@ -0,0 +1,13 @@
+AUTH_ARG_SPEC = dict(
```

Comment:
I do not think that this is needed because the args spec is defined here https://github.com/flightctl/flightctl-ansible/pull/5/files#diff-6d334f222f8b3eaccfc01ec82248c174448c457c70a161ffd1255fd1439a5b9eL19

Reply:
Ahh got it!  Just needed to resolve a couple of diffs between the docstring and those args